silkrpc/commands/engine_api.cpp

```diff
@@ -94,7 +97,23 @@ asio::awaitable<void> EngineRpcApi::handle_engine_forkchoice_updated_v1(const nl
     #ifndef BUILD_COVERAGE
     try {
     #endif
+        evmc::bytes32 empty_hash = 0x0000000000000000000000000000000000000000000000000000000000000000_bytes32;
```

Comment:
You can use `constexpr auto zero_hash = 0_bytes32;` to obtain a shorter version of this line.

Moreover, `zero_hash` seems to be a better name here because `empty_hash` is more suitable to indicate the hash of the empty ("") string